### ISSUE TITLE
Tick sound on chat message

### ISSUE THREAD

**Issue by Trainzack** [EmpiresCommunity/Unreal-Empires-Project#31]
When you receive a chat message, a subtle tick sound should play.

**Comment by Trainzack** [comment]
I'm second-guessing this decision. This seems to be trivially easy to do by adding another event to the WBP_LocalChatMessage blueprint as seen here:
![image](https://github.com/EmpiresCommunity/Unreal-Empires-Project/assets/19575360/9485ae95-5562-43bb-b9de-1157017d35de)
However, I misremembered how this was implemented in srcpires and other games. I thought srcpires plays a tick sound on every message, but it seems that only important server messages get the tick. We might want to consider this more carefully. It might even be prudent to decide whether to play the tick sound based on the relationship between the player and the sender of the message.

**Comment by RoyAwesome** [comment]
Yeah, I'm not sure.  I don't mind really subtle sound (a soft "thup" maybe?) if it's every chat message.  Maybe make it an option in the options menu?

**Comment by empZealoth** [comment]
I think we definitely should not add noise to just general chat, at least not by default. IMO Empires lives by vivid communication, and having it constantly clutter up soundscape could get pretty tiring.
 Special rare but important messages could obviously use some extra DISTINCT audio cue, but imo it should be a rare occurrence.

**Comment by RoyAwesome** [comment]
I think some options here could work.  I'd love to hear a really soft sound when someone is chatting, but I can see any chat being a problem.  I think options menu entries here are the play.  Maybe have a setting that plays this sound on [All; Team; Squad; Commander; None]

**Comment by empZealoth** [comment]
One thing I kind of worry is just excessive amount of options too. For example Factorio (especially ultra modded examples) have crazy amounts of all sorts of settings for basically everything. Which is cool. Unless you have to spend 2 hours combing through them every time you take a longer break or reinstall. 
And they also have a pretty advanced fuzzy search feature implemented too, to help players fish for possible adjustments, which is pretty indispensable with such crazy amounts of settings and would probably need to be implemented for pires at this point too.
